docs/app/Examples/views/Statistic/Types/Groups.js

```diff
@@ -2,13 +2,14 @@ import React from 'react'
 import { Statistic } from 'stardust'
 
 const { Group, Label, Value } = Statistic
+
 const items = [
-  { label: 'Faves', value: 22 },
+  { label: 'Faves', value: '22' },
   { label: 'Views', value: '31,200' },
-  { label: 'Members', value: 22 },
+  { label: 'Members', value: '22' },
```

Comment:
Hm, I suppose you are right about this.  Feel free to update here or we can merge this and make that update next.